public/contribute/building_crosswalk/android_build.md

```diff
@@ -35,13 +35,13 @@ There are two ways to define those flags:
 
 The very least you need to add to your `GYP_DEFINES` is `OS=android`. The
 default build will target 32-bit ARM; if you want to target a different
-architecture, you must also set the `target_arch` variable to `ia32`, `x64` or
```

Comment:
Like I said, some occurrences _are_ valid. `target_arch` is actually `target_arch` in gyp.